tools/daca2.py

```diff
@@ -168,14 +161,12 @@ def scanarchive(filepath, jobs, cpulimit):
     p = subprocess.Popen(cmds, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
     comm = p.communicate()
 
-    results = open('results.txt', 'at')
     if p.returncode == 0:
-        results.write(comm[1] + strfCurrTime('[%H:%M]') + '\n')
+        logging.info(comm[1] + strfCurrTime('[%H:%M]'))
```

Comment:
Will this write a [INFO] on the first warning or on all warnings? I would prefer that it is not written at all.